src/codegen/bound.rs

```diff
@@ -0,0 +1,62 @@
+use crate::{
+    analysis::{
+        bounds::{Bound, BoundType},
+        ref_mode::RefMode,
+    },
+    library::Nullable,
+};
+
+impl Bound {
+    /// Returns the type parameter reference.
+    /// Currently always returns the alias.
+    pub(super) fn type_parameter_reference(&self) -> char {
+        self.alias
+    }
+
+    /// TODO: Naming
```

Comment:
```suggestion
    // TODO: Naming
```

Reply:
The naming seems fine to me FWIW

Reply:
Cool, the `/// TODO` is entirely gone now!